### ISSUE TITLE
Rubick cannot lift Legion Commander whilst she is duelling

### ISSUE THREAD

**Issue by IvanL11** [jeffhill/Dota2#190]
### Ability name

Telekinesis 

### Description

When an enemy Legion Commander is duelling, Rubick cannot lift her with telekinesis and it also puts the spell on cooldown. 

### Example Match ID (and possibly Timestamp)

Not played it recently, tried in demo and is still prevalent in game

**Comment by Bu3nyy** [comment]
This is intended. Telekinesis was added to the list of abilities that cannot affect a Duelling unit, along with Force Staff and co.

**Comment by NMOTH** [comment]
This is intended mechanic, not a bug.

**Comment by IvanL11** [comment]
Fair enough if it's intended but why is duel only unaffected by Geo grip and telekinesis, seems kind of bizarre to me.

**Comment by arkhanov** [comment]
> This is intended. Telekinesis was added to the list of abilities that cannot affect a Duelling unit, along with Force Staff and co.

Can you lift your ally with shard?

**Comment by NMOTH** [comment]
![image](https://user-images.githubusercontent.com/106769588/171960057-ecea46c8-d7cd-4c53-a31c-1de9ccd6ab49.png)

**Comment by Uwila** [comment]
Isn't the wiki descriptive rather than prescriptive?

**Comment by jere535** [comment]
> Isn't the wiki descriptive rather than prescriptive?

Wiki mostly just states observed behaviour rather than whether something is intended or not.

**Comment by real-vaLor** [comment]
Just because this "interaction" is well-known does not mean it isn't a bug. The wiki creators are not the developers and shouldnt be used as a source for whether something is intentional or not.

**Comment by reacheight** [comment]
It's so stupid. You can hook, skewer and just stun Duelling units. Why can't you lift it..
